packages/beeq/src/components/tag/scss/bq-tag.scss

```diff
@@ -0,0 +1,101 @@
+/* -------------------------------------------------------------------------- */
+/*                        Tag styles                        */
+/* -------------------------------------------------------------------------- */
+
+@import './bq-tag.variables';
+
+:host {
+  @apply relative block;
+}
+
+:host(.is-hidden) {
+  @apply hidden;
+}
+
+.bq-tag {
+  @apply inline-flex cursor-pointer flex-row items-center justify-center bg-ui-secondary;
+
+  // Focus
+  @apply [&:not(.disabled)]:focus-visible:focus;
+
+  // Hover
+  @apply [&:not(.disabled)]:hover:bg-ui-secondary-hover;
+
+  // Active
+  @apply [&.active:not(.disabled)]:bg-ui-brand [&.active:not(.disabled)]:text-text-primary-alt;
+
+  // Disabled
+  @apply [&.disabled]:cursor-not-allowed [&.disabled]:border-none [&.disabled]:bg-ui-secondary-disabled [&.disabled]:text-text-primary-disabled;
+}
+
+.bq-tag__wrapper {
+  &--extra_small {
+    @apply gap-xs2 rounded-xs px-xs py-xs3;
+  }
+
+  &--small {
+    @apply gap-xs2 rounded-xs px-xs py-xs3;
+  }
+
+  &--medium {
+    @apply gap-xs rounded-s px-s py-xs2;
+  }
+}
```

Comment:
We can move all these to JSX, something like:

```tsx
        <button
          class={{
            'bq-tag gap-xs rounded-s px-s py-xs2 font-medium leading-regular': true,
            active: !this.disabled && this.selected,
            'gap-xs2 rounded-xs px-xs py-xs3': this.size !== 'medium',
            [`bq-tag__${this.type}__${this.variant}`]: this.hasColor,
          }}
```

it might work